gql/transport/transport.py

```diff
@@ -17,6 +20,24 @@ def execute(self, document: DocumentNode, *args, **kwargs) -> ExecutionResult:
             "Any Transport subclass must implement execute method"
         )  # pragma: no cover
 
+    @abc.abstractmethod
```

Comment:
The `execute_batch` method cannot be an `abstractmethod` here.
Doing so would break all the existing custom transports made by users which does not contain the method.

Reply:
agree, better to leave the check in runtime to avoid to always implement this